scripts/entity/world/settlements/stronghold_player_base.nut

```diff
@@ -26,10 +18,8 @@ this.stronghold_player_base <- this.inherit("scripts/entity/world/settlement", {
 		this.m.HousesMax = 2;
 		this.m.AttachedLocationsMax = 3;
```

Comment:
consider extracting these two magic numbers as well.

Reply:
Attached locations are extracted. Not sure about houses, if anything that should be a part of the (mostly unused) visual config. It's not used in the actual function either way.